src/store/reviewSpot/index.js

```diff
@@ -34,7 +30,9 @@ const state = {
     mobileError: '',
     latlongError: '',
     hasError: false,
+    hasSuccess: false,
```

Comment:
Should hasSuccess has always the inverted value of hasError.
So whenever we need to check hasSuccess we can directly check `!hasError`